### PR TITLE
Add translations to pt_BR and es

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

File: com.microsoft.Teams.metainfo.xml [hunk 1]
```diff
@@ -4,46 +4,13 @@
...
-        <p>
-            By downloading this app, you agree to the license and privacy terms (see aka.ms/privacy).
-            To learn more, please visit aka.ms/microsoftteams.
-        </p>
-    </description>
-
+    <summary>Microsoft Teams is your hub for teamwork in Office 365</summary>
+    <summary xml:lang="pt_BR">O Microsoft Teams é o hub de trabalho em equipe do Office 365</summary>
+    <summary xml:lang="es">Microsoft Teams es el concentrador de trabajo en equipo de Office 365</summary>
+    <description><p></p>All your team conversations, files, meetings, and apps live together in a single shared workspace. This app requires a paid Office 365 commercial subscription, or a free or trial subscription of Microsoft Teams. If you’re not sure about your company’s subscription or the services you have access to, visit Office.com/Teams to learn more or contact your IT department.<p>NOTE: This wrapper is not verified by, affiliated with, or supported by AnyDesk.</p></description>
```
Comment (Lctrs) [inline com.microsoft.Teams.metainfo.xml]:
```suggestion
    <description><p></p>All your team conversations, files, meetings, and apps live together in a single shared workspace. This app requires a paid Office 365 commercial subscription, or a free or trial subscription of Microsoft Teams. If you’re not sure about your company’s subscription or the services you have access to, visit Office.com/Teams to learn more or contact your IT department.<p>NOTE: This wrapper is not verified by, affiliated with, or supported by Microsoft Corporation.</p></description>
```

Here and below.

**Comment by Lctrs** [comment]
Would it be possible to improve formatting of your changes ?

File: com.microsoft.Teams.desktop [hunk 2]
```diff
@@ -2,12 +2,14 @@
...
+Comment=Microsoft Teams is your hub for teamwork in Office 365
+Comment[pt_BR]=O Microsoft Teams é o hub de trabalho em equipe do Office 365
+Comment[es]=Microsoft Teams es el concentrador de trabajo en equipo de Office 365
 Exec=teams %U
 Icon=com.microsoft.Teams
 Terminal=false
 StartupNotify=true
 StartupWMClass=Microsoft Teams - Preview
-Categories=AudioVideo;Network;
+Categories=AudioVideo;Network;Chat;Office;
```
Comment (Lctrs) [inline com.microsoft.Teams.desktop]: Please revert this changes. This `.desktop` has been extracted from the official `.deb` package. I would prefer not diverge too much from upstream.

**Comment by LeandroStanger** [comment]
bot, build

**Comment by Lctrs** [comment]
Thank you @LeandroStanger.
